duckduckgo/iOS
# onboarding experiment 2

Task/Issue URL: https://app.asana.com/0/inbox/392891325557408/1112862352963248/1116310383481169
Tech Design URL:
CC:

**Description**:

This experiment is to see the effect of an extra information button thus the control group is also the summary screen and we have released the contextual tips from experiment 1.

Check on following devices:

* SE
* iPhone 8
* iPhone X 
* iPad

You'll also need to update your local fonts to include extra bold.  Check Team folder in Resilio.

**Steps to test this PR**:

Variant `sc` control:
1. No second button on boarding
1. Confirm contextual tips shown after dismissing onboarding

Variant `mr`:
1. Shows explore privacy button
1. Tap it to launch a web container which shows an app info page (if not merged yet will show a search result for `app_info`
1. Dismiss to return to onboarding
1. Confirm contextual tips shown after dismissing onboarding

Variant `mq`:
1. Shows customize settings button
1. Tap it to launch settings
1. Dismiss to return to onboarding
1. Check contextual tips shown appropriately even if initial layout is changed to a centered search

Pixels:

* `mo` - fired when onboarding is shown
* `mo_s` - fired first time only when customize settings button pressed
* `mo_e` - fired first time only when explore privacy button pressed

Home Row CTA

* This has now been deferred to a day that is not the current installation day. Change date on device and relaunch app to show it.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
